### ISSUE TITLE
MultipleRuns with repeated strategy creates only one instance

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [pedro-andrade-inpe/calibration#115]
The repeated strategy is instantiating the model only once (it is implemented before the loop).  In the way it is implemented now all the simulations will produce the same output, as it will simulate only in the first execution. The other executions will not have any Event to be executed. Implement a test that uses a Model based on random numbers with output function to check that the results are different in each simulation.

**Comment by agojunior** [comment]
was not really been able to fix bus #115 , everytime I try to fix this, this bug happens:
http://i.imgur.com/Z5I4wBo.png

**Comment by pedro-andrade-inpe** [comment]
You can use function clone below. It will be added to the next version of TerraME, but for now you should put it as a local function in calibration package.

```lua
function clone(mtable)
    mandatoryArgument(1, "table", mtable)

    local result = {}

    forEachElement(mtable, function(idx, value, mtype)
        if mtype == "table" then
            result[idx] = clone(value)
        else
            result[idx] = value
        end
    end)

    return result
end
```
